packages/cozy-procedures/src/components/documents/DocumentHolder.jsx

```diff
@@ -1,33 +1,73 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { translate, Chip, Icon } from 'cozy-ui/transpiled/react/'
+import {
+  translate,
+  Chip,
+  Icon,
+  Modal,
+  SubTitle,
+  Text
+} from 'cozy-ui/transpiled/react/'
 import { CozyFile } from 'cozy-doctypes'
 
 import DocumentsDataFormContainer from '../../containers/DocumentsDataForm'
 
-const DocumentHolder = ({ document, unlinkDocument, documentId }) => {
-  const splittedName = CozyFile.splitFilename(document)
-  return (
-    <Chip variant="dashed" className="u-w-100">
-      <Icon icon={`file-type-${document.class}`} size={24} className="u-mr-1" />
-      <span className="u-flex-grow-1 u-ellipsis">
-        {splittedName.filename}
-        <span className="u-coolGrey">{splittedName.extension}</span>
-      </span>
-      <Icon
-        icon="cross"
-        size={16}
-        className="u-pr-1 u-c-pointer"
-        onClick={() => unlinkDocument({ document, documentId })}
-      />
-    </Chip>
-  )
+class DocumentHolder extends Component {
+  state = {
+    isModalOpened: false
+  }
+
+  render() {
+    const { document, unlinkDocument, documentId, t } = this.props
+    const { isModalOpened } = this.state
```

Comment:
maybe be more precise about the modal : `isUnlinkConfirmationModalOpened` is a bit long though ;-)